### PR TITLE
Update user guide

### LINKED ISSUE #23: Update `User Guide (UG)`
[Refer to CS2103T Textbook for exact specifications](https://nus-cs2103-ay2122s2.github.io/website/schedule/week7/project.html#3-update-the-ug)

### PR DESCRIPTION
**Pull request by Ryan-L98**
Fixes #23.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,11 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+TeachWhat is a **desktop app for tutors to manage their schedule, students, classes, and
```
Comment (nnmoq) [inline docs/UserGuide.md:6]:
```suggestion
TeachWhat! is a **desktop app for tutors to manage their schedule, students, classes, and 
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,7 +3,11 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+TeachWhat is a **desktop app for tutors to manage their schedule, students, classes, and 
+income**, optimized for use via a Command Line Interface (CLI) while still having the 
+benefits of a Graphical User Interface (GUI). 
+If you can type fast, TeachWhat can get your tuition management tasks done faster than
```
Comment (nnmoq) [inline docs/UserGuide.md:9]:
```suggestion
If you can type fast, TeachWhat! can get your tuition management tasks done faster than 
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `TeachWhat.jar` from [here](https://github.com/AY2122S2-CS2103T-W11-3/tp/releases).
```
Comment (nnmoq) [inline docs/UserGuide.md:21]:
```suggestion
2. Download the latest `TeachWhat!.jar` from [here](https://github.com/AY2122S2-CS2103T-W11-3/tp/releases).
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`ls -s`** : Lists all students.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`add -c -r programming 101 -d 2022-02-17 -t 1700`** : 
+   Adds a recurring class programming 101 every Thursday at 5pm from 17th Feb 2022.
 
-   * **`clear`** : Deletes all contacts.
+   * **`save TeachWhat/store.txt`** : Saves students and classes into the store.txt text file.
```
Comment (nnmoq) [inline docs/UserGuide.md:38]:
```suggestion
   * **`save TeachWhat!/store.txt`** : Saves students and classes into the store.txt text file.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+Example: `save /TeachWhat/store.txt` sets the save file to be 
+`store.txt` in the directory `TeachWhat`.
+
+---
 
-### Archiving data files `[coming in v2.0]`
+### Exit
 
-_Details coming soon ..._
+Closes TeachWhat window.
```
Comment (nnmoq) [inline docs/UserGuide.md:174]:
```suggestion
Closes TeachWhat! window.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+Format: `quit`
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
+the data of your previous TeachWhat folder.
```
Comment (nnmoq) [inline docs/UserGuide.md:184]:
```suggestion
the data of your previous TeachWhat! folder.
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+| **Delete Student** | `rm -s [STUDENT_ID]`<br> e.g. `rm -s 13`                                                                       |
+| **Delete Class**   | `rm -c [CLASS_ID]`<br> e.g.`rm -c 5`                                                                           |
+| **Assign Student** | `add -s -c [STUDENT_ID] [CLASS_ID]`<br> e.g.`add -s -c 5 11`                                                   |
+| **List Students**  | `ls -s`                                                                                                        |
+| **List Class**     | `ls -c`                                                                                                        |
+| **View Student**   | `student [STUDENT_ID]` <br> e.g. `student 5`                                                                   |
+| **View Class**     | `class [CLASS_ID]`<br> e.g. `class 2`                                                                          |
+| **Save**           | `save [RELATIVE_PATH]` <br> e.g. `/TeachWhat/store.txt`                                                        |
```
Comment (nnmoq) [inline docs/UserGuide.md:201]:
```suggestion
| **Save**           | `save [RELATIVE_PATH]` <br> e.g. `/TeachWhat!/store.txt`                                                        |
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -14,179 +18,185 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+___
+
+### Setting a save file location
+Sets the file to retrieve from and save data to, to the specified directory. 
+If the file does not exist, a new file will be created.
+
+Format: `save [RELATIVE_PATH]`
+
+Example: `save /TeachWhat/store.txt` sets the save file to be 
+`store.txt` in the directory `TeachWhat`.
```
Comment (nnmoq) [inline docs/UserGuide.md:168]:
```suggestion
Example: `save /TeachWhat!/store.txt` sets the save file to be 
`store.txt` in the directory `TeachWhat!`.
```

**Review by nnmoq — CHANGES_REQUESTED** [review]
Try to keep the name consistent. TeachWhat!

**Review by jamesyeap — CHANGES_REQUESTED** [review]

**Review by nnmoq — APPROVED** [review]

**Review by jamesyeap — APPROVED** [review]

**Pull request merged**